spec/cc/engine/analyzers/php/main_spec.rb

```diff
@@ -2,12 +2,17 @@
 require 'cc/engine/analyzers/php/main'
 require 'cc/engine/analyzers/reporter'
 require 'cc/engine/analyzers/engine_config'
+require 'cc/engine/analyzers/file_list'
 require 'flay'
 require 'tmpdir'
 
 module CC::Engine::Analyzers::Php
   describe Main do
-    before { @code = Dir.mktmpdir }
+    before do
+      @code = Dir.mktmpdir
+      Dir.chdir(@code)
```

Comment:
Yeah, I think the migration to RSpec would be more valuable than pulling in `minitest/around` here since this feels a lot like RSpec but has a lot of quirks that catch you by surprise.

Can we keep this as is for now and in the next PR I'll migrate to RSpec so we can have `around` and fix this issue?